src/index.css

```diff
@@ -28,6 +29,7 @@
   --secondary-light-transparent: #ff7bed69;
   --accent-light-active-transparent: #fed8239e;
   --secondary-light-active-transparent: #ff7bed9e;
+  --formal-accent-light-transparent: #efecd6cc;
```

Comment:
Can you convert those colors `alpha` channel into more understandable form, please? Here's the table: https://gist.github.com/lopspower/03fb1cc0ac9f32ef38f4

Reply:
It's a correct hex color with alfa. There is no need to change it